src/routes/Landing.js

```diff
@@ -35,12 +35,17 @@ class Landing extends Component {
             ...params,
             isModalOpen: params && Object.keys(params).length > 0
         });
-
+        
```

Comment:
There is a trailing space here that is causing Travis to fail the build

Reply:
fixed!